apps/dex/src/compounds/Margin/Trade.tsx

```diff
@@ -20,7 +20,7 @@ function HtmlUnicode({ name }: { name: string }) {
     RightwardsArrow: "&rightarrow;", // https://www.compart.com/en/unicode/U+2192
   };
   const entity = unicodes[name] || `MISSING_UNICODE: ${name}`;
-  return <span dangerouslySetInnerHTML={{ __html: entity }} />;
+  return null;
```

Comment:
What's the reason behind this?

Reply:
Oops, was debugging, must have committed by mistake